<?xml version="1.0" encoding="UTF-8"?>
<!--The XML tag is used for nano and gedit colors -->

<!-- This launch file starts the motor_hat_twist_node in order to be controlled 
by the ROS controll android application -->
<launch>

    <node name="motor_hat_twist_node" pkg="istia_rover" type="motor_hat_twist_node" output="screen">
        <param name="motor_drive" value="0"/>
        <param name="max_speed" value="80"/>
        <param name="max_twist" value="1"/>
        <param name="topic_name" value="/joy_teleop/cmd_vel"/>
    </node>

</launch>
